keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1814,18 +1759,14 @@ No
 
 It is not expected to...
 
-### Troubleshooting
-
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
+We do not currently have any apples-to-apples comparisons; the
+`nftables` perf job uses more CPU than the corresponding `iptables`
+job, but this is because it doesn't run with `minSyncPeriod: 10s` like
+the `iptables` job does, and so it syncs rule changes more often.
```

Comment:
But we could test this with the same config? Why not?

Reply:
Because the config used by the iptables test is objectively incorrect. :stuck_out_tongue_closed_eyes: 

I suppose I should try to get that fixed

Reply:
https://kubernetes.io/docs/reference/networking/virtual-ips/#minimize-iptables-restore